6281.97 <c-2#36> Created logger
6281.97 <c-2#36> appl_initialize was called
6281.97 <c-2#36:1> Logger was updated
6281.97 <c-2#36:1> Started intializing cube lib
6281.97 <c-2#36:1> Finished intializing cube lib with worker, starting to work
6281.98 <c-2#36:1> requestCubes => [24] Buffer size: 0
6281.98 <c-2#36:1> Sent requestCubes signal to root
6281.98 <c-2#36:1> Received 1 cubes from root
6281.98 <c-2#36:1> The main loop continues
6281.98 <c-2#36:1> Started solving a cube
6281.98 <c-2#36:1> Cube failed
6281.98 <c-2#36:1> Added failed cube
6281.99 <c-2#36:1> returnFailedCubes => [24] Buffer size: 3
6281.99 <c-2#36:1> Sent 1 failed cubes to root
6281.99 <c-2#36:1> Received receivedFailedCubes signal from root
6282 <c-2#36:1> requestCubes => [24] Buffer size: 0
6282 <c-2#36:1> Sent requestCubes signal to root
6282 <c-2#36:1> Received 1 cubes from root
6282 <c-2#36:1> The main loop continues
6282 <c-2#36:1> Started solving a cube
6694.54 <c-2#36:1> appl_interrupt was called
6694.54 <c-2#36:1> Interrupting worker
6694.54 <c-2#36:1> appl_interrupt was called
6694.54 <c-2#36:1> appl_withdraw was called
6694.54 <c-2#36:1> Started withdraw thread
6694.54 <c-2#36:1> STOP (0.00s since last cb)
6694.54 <c-2#36:1> Solving interrupted
6694.54 <c-2#36:1> The main loop continues
6694.54 <c-2#36:1> Leaving the main loop due to an interruption
6694.54 <c-2#36:1> Finished withdrawing lib
6694.54 <c-2#36:1> Enter destructor of CubeWorker
6694.54 <c-2#36:1> Time waiting for messages: 0.001
6694.55 <c-2#36:1> Enter destructor of CubeWorkerInterface
6694.55 <c-2#36:1> Enter destructor of CubeRoot
6694.55 <c-2#36:1> Finished withdraw thread
6755.56 <c-2#36:1> Enter destructor
6755.56 <c-2#36:1> Time working: 412.569
6755.56 <c-2#36:1> Time suspended: 0.000
6755.56 <c-2#36:1> Joined cleanup thread
6755.56 <c-2#36:1> Exit destructor
6755.56 <c-2#36:1> Destructing logger
